code/modules/mob/living/simple_animal/posessed_object.dm

```diff
@@ -93,10 +103,14 @@
 		qdel(src)
 
 	possessed_item = loc
-	forceMove( possessed_loc )
+	forceMove(possessed_loc)
 	possessed_item.forceMove(src) // We'll keep the actual item inside of us until we die.
 
+
 	update_icon(1)
```

Comment:
uhhhhh. This means to only update the name... I doubt that is correct.
```suggestion
	update_icon()
```

Reply:
There is an overriden `update_icon` in this case, it does not behave as the rest of the `update_icon` calls do

Reply:
Oh I see yeah. Aye should be fine then indeed